test/unit/src/common/actions/t_repository-input.js

```diff
@@ -54,9 +54,9 @@ describe('repository input actions', () => {
       expect(store.getActions()).toInclude(expectedAction);
     });
 
-    it('should create a valid flux standard action', () => {
-      expect(isFSA(action)).toBe(true);
-    });
+    // it('should create a valid flux standard action', () => {
+    //   expect(isFSA(action)).toBe(true);
+    // });
```

Comment:
Include a comment about why this test is disabled?  (And maybe use `xit` or similar instead.)